### PR TITLE
Bug 1174380 - Hide the hangup button when receiving a second call and properly align the ignore button r=drs

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=15f99005c51e8702db2c801c23d91e42463fe0f9

File: apps/callscreen/test/unit/call_screen_test.js [hunk 1]
```diff
@@ -779,10 +785,10 @@ suite('call screen', function() {
 
     test('should remove class of callToolbar and incomingContainer',
```
Comment (DouglasSherk) [inline apps/callscreen/test/unit/call_screen_test.js]: ```s/callToolbar/callOptions/``` .

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b92fd2f83ad84815456d114829eb2c9aeddaab27

File: apps/callscreen/test/unit/call_screen_test.js (hunk 1 above)
Comment (gabrielesvelto) [inline apps/callscreen/test/unit/call_screen_test.js]: Good catch, I'll fix it and merge

**Pull request merged**